orbit/estimators/stan_estimator.py

```diff
@@ -252,14 +254,15 @@ def fit(
 
         # `stan.optimizing` automatically returns all defined parameters
         # filter out unnecessary keys
-        posteriors = {param: stan_extract[param] for param in model_param_names}
+        posteriors = {
+            param: stan_extract[param] for param in model_param_names + ["loglk"]
+        }
         training_metrics = dict()
 
-        # extract `log_prob` in addition to defined model params
-        # this is for the BIC calculation
+        # loglk is needed for BIC calculation
         training_metrics.update({"loglk": stan_extract["loglk"]})
-        # FIXME: this needs to be the full length of all parameters instead of the one we sampled?
-        # FIXME: or it should be not include latent varaibles / derive variables?
+        # TODO: this needs to be the full length of all parameters instead of the one we sampled?
+        # TODO: or it should be not include latent variables / derive variables?
         training_metrics.update({"num_of_params": len(model_param_names)})
```

Comment:
nit: same as above. Are there any reasons we're doing this way (i.e., creating a dict in multiple steps)?